### PR TITLE
Add SimbaChain swag

### LINKED ISSUE #196: Add SimbaChain swag
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : SimbaChain

Q. Where have you seen the announcement (URL)?  
A. : https://simbachain.com/community/join-our-community/

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing (t-shirt)

Q. Have you received swag from this?  
A. : No

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Yes

Comes from this discussion: https://github.com/ethereum/go-ethereum/issues/2104#issuecomment-399721639

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by vivekptnk**
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate pull request.
- [x] I've checked that this is linked to a new swag opportunity issue <!>

Implement swag opportunity: SimbaChain

Closes #196

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -232,5 +232,16 @@
...
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Sign up with your emailID to recieve a ",
+    "reference": " https://simbachain.com/community/join-our-community/",
+    "image": "https://simbachain.com/wp-content/uploads/2018/07/SIMBA-Blockchain-Shirt-V2.png",
+    "dateAdded": "2018-12-07T5:58:05.000Z",
+    "tags": ["clothing"]
   }
+
```
Comment (vikaspotluri123) [inline data.json]: This extra whitespace isn't necessary

File: data.json [hunk 2]
```diff
@@ -232,5 +232,16 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Sign up with your emailID to recieve a ",
```
Comment (vikaspotluri123) [inline data.json]: The description doesn't seem complete 😉

Comment (vivekptnk) [reply data.json]: Noted and Fixed

File: data.json [hunk 3]
```diff
@@ -232,5 +232,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Register and get a free Blockchain Shirt!",
```
Comment (aslafy-z) [inline data.json]: It would be great to add a mention about the block chain controversy (eth_listtransactions) => look at the issue for references

Comment (vivekptnk) [reply data.json]: just a comment about the the controversy?

Comment (aslafy-z) [reply data.json]: Look at the second link of the issue

File: package-lock.json [hunk 4]
```diff
@@ -1059,12 +1059,6 @@
         "ansi-wrap": "0.1.0"
```
Comment (aslafy-z) [inline package-lock.json]: This file shouldn't appear in this PR

Comment (vivekptnk) [reply package-lock.json]: Im going to close this PR and redo the changes

**Pull request closed without merging**